flintrock/ec2.py

```diff
@@ -283,6 +283,47 @@ def check_network_config(*, region_name: str, vpc_id: str, subnet_id: str):
         )
 
 
+def get_ec2_security_groups(
+        vpc_id,
+        region,
+        security_groups) -> "List[boto3.resource('ec2').SecurityGroup]":
+    """
+    Get security groups with the given security group names. Also accepts security
+    group ids.
+    """
+    ec2 = boto3.resource(service_name='ec2', region_name=region)
+
+    # Resolve security group names
```

Comment:
I think we can drop all the comments like this one since the code is pretty self-explanatory.

In general, I prefer to add comments to explain _why_ something was done when it may not be obvious, as opposed to _what_ was done which is usually clear enough from the code.